src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -62,9 +63,18 @@ export default View.extend({
     }
 
     links.forEach(link => {
-      this.add(Link, {
-        options: link,
-      });
+      if (link.linkInfo) {
+        this.add(ToggleTextLink, {
+          options : {
+            link: new Link({ options : link }),
+            linkInfo: link.linkInfo,
+          }
+        });
+      } else {
```

Comment:
can't we just add `ToggleTextLink` and remove `else` part, because it only adds simple `Link` when `linkInfo` is not available ?